backend/webpack.config.js

```diff
@@ -64,7 +64,7 @@ const config = {
   plugins: [
     new CopyPlugin([
       { from: '../frontend/dist/', to: 'media/', force: true },
-      { from: '../LICENSE', to: '../LICENSE', toType: "file", force: true },
+      { from: '../LICENSES', to: '../LICENSES', toType: "file", force: true },
```

Comment:
{ from: '../LICENSE', to: 'LICENSE/', force: true },
      { from: '../README.md', to: 'README.md', toType: "file", force: true }

Reply:
done